JustLearn.Services.Api/Models/Profiles/Profile.cs

```diff
@@ -0,0 +1,28 @@
+﻿// -----------------------------------------------------------------------
+// Copyright (c) MumsWhoCode. All rights reserved.
+// -----------------------------------------------------------------------
+
+using System;
+using JustLearn.Services.Api.Models.Users;
+
+namespace JustLearn.Services.Api.Models.Profiles
+{
+    public class Profile
+    {
+        public Guid Id { get; set; }
+        public string Name { get; set; }
+        public ProfileType Type { get; set; }
+        public ProfileStatus Status { get; set; }
+        public DateTimeOffset ExpirationDate { get; set; }
+        public int Renewed { get; set; }
```

Comment:
If you are thinking about count fo renewals then use the name 'RenewalCount'. The word 'Renewed' reads like - Renewed or Not.